treeherder/etl/common.py

```diff
@@ -27,10 +30,11 @@ def make_request(url, method='GET', headers=None, timeout=30, **kwargs):
     return response
 
 
-def fetch_json(url, params=None):
+def fetch_json(url, params=None, headers={}):
+    headers["Accept"] = "application/json"
```

Comment:
Even though we handle Github API's headers down in `make_request` this is nice to have in the future if someone wants to call `fetch_json` and pass some other headers.